library/src/data/index.js

```diff
@@ -1 +1,24 @@
-window.CoNDeT.data = {}
\ No newline at end of file
+window.CoNDeT.data = {
+  State: (function () {
+    function constructor() {}
+
+    constructor.prototype.state = [];
+    constructor.prototype.subscribers = [];
+
+    constructor.prototype.methods = {
+      subscribe: function (fun) {
+        this.subscribers.push(fun);
+      },
```

Comment:
Metoda powinna być zapisana jako:
```
constructor.prototype.subscribe = function (fun) {
  this.subscribers.push(fun);
};
```
Tak jak jest teraz nie zadziała. Analogicznie z innymi metodami.
Można by zmienić nazwę z `fun` na `cb` albo `subscriber`. Wydaje mi się, że lepiej opisują o co chodzi
Do tego jak teraz myślę, to w tej metodzie też należy wywołać podany callback podając mu obecny state